samples/nrf_rpc/entropy_nrf53/cpunet/sysbuild.cmake

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
+
+# Include app core image if enabled
+if(SB_CONFIG_SOC_NRF5340_CPUNET)
```

Comment:
Empty app_core is not a target for app core in this sample

Reply:
How does someone run it then?

Reply:
Oh the app and net should run for the same image, that is confusing, will update...

Reply:
@KAGA164 have reworked this, please check again

Reply:
Updated